### ISSUE TITLE
Is i3 class really needed?

### ISSUE THREAD

**Issue by drgrigg** [irontigran/john-w-campbell_islands-of-space#18]
You have this defined in CSS but I don't think it's necessary. The blockquote should apply sufficient styling.

**Comment by drgrigg** [comment]
Sorry, you should actually include the poetry styling from the semantics manual.

**Comment by irontigran** [comment]
I did _attempt_ to follow the poetry section in how I styled the poem. I chose the i3 class because it matched the same amount of indentation PG had in their transcription.

What needs to be changed for the poem?

**Comment by drgrigg** [comment]
You'll need to include this in the CSS, from the semantics manual. I have removed one selector from that example because it's not needed in your specific case:

```
[epub|type~="z3998:poem"] p{
	text-align: left;
	text-indent: 0;
}

[epub|type~="z3998:poem"] p > span{
	display: block;
	text-indent: -1em;
	padding-left: 1em;
}

[epub|type~="z3998:poem"] p > span + br{
	display: none;
}

[epub|type~="z3998:poem"] + p{
	text-indent: 0;
}
```

There's no need to indent the lines any further than the default given here. i1, i2, i3, etc are used when subsequent lines need further indenting than the first line, as it common in poetry. But Collerige's lines in _The Rime of the Ancient Mariner_ aren't indented. 

Good catch on tagging this as non-fiction, by the way, since it's a real poem being quoted.

**Comment by irontigran** [comment]
I see - I wasn't really understanding how CSS was supposed to be included. Fixed here: 9e9f0e4e6f8648d73059757fbb116b69dc8b6e22
